frontend/src/types/LogRecordTypes.ts

```diff
@@ -21,3 +21,5 @@ export type GetLogRecordsReponse = {
 export type GetLogRecordCountResponse = {
   numResults: number;
 } | null;
+
+export type PostLogRecordsResponse = any | null;
```

Comment:
Can this any also be removed/replaced, it just gets caught by the linter

Reply:
I legit gaslit myself into thinking I filled out that type already 💀